ServerTests/ModelsTests/MyHttpRequestTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using FluentAssertions;
+using Moq;
+using Server.Clients;
+using Server.Exceptions;
+using Server.Models;
+using System;
+using System.Text;
+using Xunit;
+
+namespace WebServerTests.ModelsTests
+{
+    public class MyHttpRequestTests
+    {
+        [Fact]
+        public void Build_Should_Throw_InvalidRequestExceptions_When_Byte_Array_Empty()
```

Comment:
why here you have "Throw_InvalidRequestExceptions" in the test name, but in another cases you have just "Throw"?